pkg/types/plugin.go

```diff
@@ -9,11 +9,12 @@ const (
 	OCR2VRF OCR2PluginType = "ocr2vrf"
 
 	// TODO: sc-55296 to rename ocr2keeper to ocr2automation in code
-	OCR2Keeper    OCR2PluginType = "ocr2automation"
-	Functions     OCR2PluginType = "functions"
-	Mercury       OCR2PluginType = "mercury"
-	LLO           OCR2PluginType = "llo"
-	GenericPlugin OCR2PluginType = "plugin"
+	OCR2Keeper     OCR2PluginType = "ocr2automation"
+	Functions      OCR2PluginType = "functions"
+	Mercury        OCR2PluginType = "mercury"
+	LLO            OCR2PluginType = "llo"
+	GenericPlugin  OCR2PluginType = "plugin"
+	OCR3Capability OCR2PluginType = "ocr3-capability"
```

Comment:
Do we have to use a new plugin type? Can't it be GenericPlugin with a special plugin config field?

Reply:
Yes, we do -- pluginType is used in two different ways: 1) to specify the branch we'll go down in OCR2 delegate -- this will be unchanged (i.e. set to GenericPlugin), and 2) to fetch the right providerType (this will need to be set to ocr3-capability)